docs/src/examples.md

```diff
@@ -0,0 +1,103 @@
+* ``I^f`` is the [center-to-face reconstruction operator](https://clima.github.io/ClimaCore.jl/stable/operators/#ClimaCore.Operators.InterpolateC2F), called `first_order_Ic2f` in the example code.
+  - Currently this is just the arithmetic mean, but we will need to use a weighted version with stretched vertical grids.
+* ``U^f`` is the [center-to-face upwind product operator](https://clima.github.io/ClimaCore.jl/stable/operators/#ClimaCore.Operators.Upwind3rdOrderBiasedProductC2F), called `third_order_upwind_c2f` in the example code
+  - This operator is of third-order of accuracy (when used with a constant vertical velocity and some reduced, but still high-order for non constant vertical velocity).
+
+
+#### Differentiation operators
+
+- ``D_h`` is the [discrete horizontal weak spectral divergence](https://clima.github.io/ClimaCore.jl/stable/operators/#ClimaCore.Operators.WeakDivergence), called `hwdiv` in the example code.
+- ``D^c_v`` is the [face-to-center vertical divergence](https://clima.github.io/ClimaCore.jl/stable/operators/#ClimaCore.Operators.DivergenceF2C), called `vdivf2c` in the example code.
+  - This example uses advective fluxes equal to zero at the top and bottom boundaries.
+- ``G_h`` is the [discrete horizontal spectral gradient](https://clima.github.io/ClimaCore.jl/stable/operators/#ClimaCore.Operators.Gradient), called `hgrad` in the example code.
+
+To discretize the hyperdiffusion operator, ``g(\rho, q) = - \nu_4 [\nabla^4 (\rho q)]``, in the horizontal direction, we compose the horizontal weak divergence, ``D_h``, and the horizontal gradient operator, ``G_h``, twice, with an intermediate call to [`weighted_dss!`](@ref) between the two compositions, as in ``[g_2(\rho, g) \circ DSS(\rho, q) \circ g_1(\rho, q)]``, with:
+- ``g_1(\rho, q) = D_h(G_h(q))``
+- ``DSS(\rho, q) = DSS(g_1(\rho q))``
+- ``g_2(\rho, q) = -\nu_4 D_h(\rho G_h(\rho q))``
+    - with ``\nu_4`` is the hyperviscosity coefficient
+
+#### Flow and set-up
+
+This test case is set up in a Cartesian (box) domain `` [-2 \pi, 2 \pi]^2 \times [0, 4 \pi] \textrm{m}^3``, doubly periodic in the horizontal direction, but not in the vertical direction.
+
+The flow was chosen to be a spiral, i.e., so to have a horizontal uniform rotation, and a vertical velocity ``\boldsymbol{u}_v \equiv w = 0`` at the top and bottom boundaries and ``\boldsymbol{u}_v \equiv w = 1`` in the center of the domain. Moreover, the flow is reversed in all directions halfway through the period so that the tracer blobs go back to its initial configuration (using the same speed scaling constant which was derived to account for the distance travelled in all directions in a half period).
+
+```math
+\begin{equation}
+    u = -u0 * (y - c_y) * cos(\pi t / T_f)
+    v = u0 * (x - c_x) * cos(\pi t / T_f)
+    w = u0 * sinpi(z / z_m) * cos(\pi t / T_f)
+\label{eq:flow}
+\end{equation}
```

Comment:
@simonbyrne , I added the docs for this example. I am planning to do that for other examples I worked on as well. 
I have a question. It seems like the `\equation` or `\align` environments are not parsed correctly by Documenter.jl. But I have seen that it was used successfully in the Microphysics package, see. e.g., this [line](https://github.com/CliMA/CloudMicrophysics.jl/blob/3bb630d06005d61f42e2d908c3169944576f9abb/docs/src/Microphysics_1M.md?plain=1#L159). Is there a package/sub-package dependency we need to have it working here as well?
Thanks!

Reply:
I tried to define `mathengine` similarly to what I've seen in the CloudMicrophysics.jl [docs](https://github.com/CliMA/CloudMicrophysics.jl/blob/3bb630d06005d61f42e2d908c3169944576f9abb/docs/make.jl#L19) but there would be an error `ERROR: LoadError: UndefVarError: MathJax not defined` in that case. So I am not sure how else make the `equation` environment and its labels work